tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -99,16 +97,11 @@ def _add_host_config(self):
                     }
                 }
             }
+        if mode == FUNCTION_EVENT_TRIGGER_MODE:
+            config['extensionBundle'] = {"id": "Microsoft.Azure.Functions.ExtensionBundle",
+                                         "version": "[1.*, 2.0.0)"}
```

Comment:
Is it worth pulling this version value out to the `Constants` file?

Reply:
I think if we pull version from constants, we should pull all other config values as well.. What do you think?

Reply:
I would be fine with that.